src/src/pages/Dashboard/components/AimIntegrations/AimIntegrations.tsx

```diff
@@ -52,15 +60,22 @@ trainer = Trainer(
     {
       title: 'Integrate Keras & tf.keras',
       docsLink: DOCUMENTATIONS.INTEGRATIONS.KERAS,
-      code: `import aim
+      code: `import mlflow
+from mlflow import set_tracking_uri
+import mlflow.keras
+
+# Set FastTrackML tracking server
+set_tracking_uri("${fasttrack_server}")
```

Comment:
This would be more Pythonic:
```suggestion
      code: `import mlflow
import mlflow.keras

# Set FastTrackML tracking server
mlflow.set_tracking_uri("${fasttrack_server}")
```